test/e2e/lib/build.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lib
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+
+	kngcptesting "github.com/google/knative-gcp/pkg/reconciler/testing"
+	"github.com/google/knative-gcp/test/e2e/lib/resources"
+	"google.golang.org/api/cloudbuild/v1"
```

Comment:
I would prefer the cloud.google.com/go/cloudbuild/apiv1/v2 client

Reply:
There is no v2 client https://godoc.org/google.golang.org/api/cloudbuild

Reply:
That's a different package: https://pkg.go.dev/cloud.google.com/go/cloudbuild/apiv1/v2?tab=doc.

Reply:
done